lib/Entity.js

```diff
@@ -40,6 +43,8 @@ export class Entity {
      * @property {number} rotation - The rotation of the entity.
      * @property {number} x - The x-coordinate of the centre of the entity.
      * @property {number} y - The y-coordinate of the centre of the entity.
+     * @property {number} ageless - Whether the object ages/decrements it's lifespan.
```

Comment:
While a cool property/idea, we want to keep the number of properties low. 
I don't think ageless offers enough bank for buck considering it "costs" us adding another user facing property sadly.

Reply:
My thoughts on lifespan.

If lifespan has not be set (it has no lifespan), then it should be null or undefined. In user code I wouldn't want to have to compare lifespan against infinity to check if something has a lifespan, that seems messy.

Then at the start of frame in update (the proper update before user code), you decrement the lifespan and check if it's less than 0, if it is, lifespan is set to 0.

User code can then check objects if the lifespan is 0 to run other code (since we don't have an event system that can listen for an object dying) and finally in the cleanup code after user code, we can delete the object if it's lifespan is less than 0.

Reply:
Made the suggested changes.